Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -20,8 +20,8 @@ To get started with the Mixed Reality Toolkit you will need:
   when building for OpenVR.
 
 
-# MR Learning Base Module Tutorial 
-If you are new to MRTK, or MR development, we recommend you check out the [MR Learning Base Module](https://docs.microsoft.com/en-us/windows/mixed-reality/mrlearning-base) which uses MRTK v2.
+# Getting started tutorials
+If you are new to MRTK, or MR development, we recommend you check out the [Getting started tutorials](https://docs.microsoft.com/en-us/windows/mixed-reality/mrlearning-base) which uses MRTK v2.
```

Comment:
The link refers to HL2 ONLY documentation, which on the front page specifically excludes HL1 and Immerive apps.  Is this the best first experience landing page for "Getting Started"?